core_impl/src/test/java/io/opencensus/stats/ViewManagerImplTest.java

```diff
@@ -71,391 +92,368 @@
-//    clock.setTime(Timestamp.create(3, 3));
-//    DistributionViewData viewData1 = (DistributionViewData) viewManager.getView(VIEW_NAME);
-//    clock.setTime(Timestamp.create(4, 4));
-//    DistributionViewData viewData2 = (DistributionViewData) viewManager.getView(VIEW_NAME_2);
-//    assertThat(viewData1.getStart()).isEqualTo(Timestamp.create(1, 1));
-//    assertThat(viewData1.getEnd()).isEqualTo(Timestamp.create(3, 3));
-//    assertDistributionAggregatesEquivalent(viewData1.getDistributionAggregates(), expectedAggs);
-//    assertThat(viewData2.getStart()).isEqualTo(Timestamp.create(2, 2));
-//    assertThat(viewData2.getEnd()).isEqualTo(Timestamp.create(4, 4));
-//    assertDistributionAggregatesEquivalent(viewData2.getDistributionAggregates(), expectedAggs);
-//  }
-//
-//  @Test
-//  public void testMultipleViewsDifferentMeasures() {
-//    MeasureDouble measure1 =
-//        Measure.MeasureDouble.create(MEASURE_NAME, MEASURE_DESCRIPTION, MEASURE_UNIT);
-//    MeasureDouble measure2 =
-//        Measure.MeasureDouble.create(MEASURE_NAME_2, MEASURE_DESCRIPTION, MEASURE_UNIT);
-//    View view1 =
-//        createDistributionView(
-//            VIEW_NAME, measure1, DISTRIBUTION_AGGREGATION_DESCRIPTOR, Arrays.asList(KEY));
-//    View view2 =
-//        createDistributionView(
-//            VIEW_NAME_2, measure2, DISTRIBUTION_AGGREGATION_DESCRIPTOR, Arrays.asList(KEY));
-//    clock.setTime(Timestamp.create(1, 0));
-//    viewManager.registerView(view1);
-//    clock.setTime(Timestamp.create(2, 0));
-//    viewManager.registerView(view2);
-//    statsRecorder.record(
-//        createContext(factory, KEY, VALUE),
-//        MeasureMap.builder().set(measure1, 1.1).set(measure2, 2.2).build());
-//    clock.setTime(Timestamp.create(3, 0));
-//    DistributionViewData viewData1 = (DistributionViewData) viewManager.getView(VIEW_NAME);
-//    clock.setTime(Timestamp.create(4, 0));
-//    DistributionViewData viewData2 = (DistributionViewData) viewManager.getView(VIEW_NAME_2);
-//    assertThat(viewData1.getStart()).isEqualTo(Timestamp.create(1, 0));
-//    assertThat(viewData1.getEnd()).isEqualTo(Timestamp.create(3, 0));
-//    assertDistributionAggregatesEquivalent(
-//        viewData1.getDistributionAggregates(),
-//        Arrays.asList(
-//            StatsTestUtil.createDistributionAggregate(
-//                Arrays.asList(Tag.create(KEY, VALUE)), BUCKET_BOUNDARIES, Arrays.asList(1.1))));
-//    assertThat(viewData2.getStart()).isEqualTo(Timestamp.create(2, 0));
-//    assertThat(viewData2.getEnd()).isEqualTo(Timestamp.create(4, 0));
-//    assertDistributionAggregatesEquivalent(
-//        viewData2.getDistributionAggregates(),
-//        Arrays.asList(
-//            StatsTestUtil.createDistributionAggregate(
-//                Arrays.asList(Tag.create(KEY, VALUE)), BUCKET_BOUNDARIES, Arrays.asList(2.2))));
-//  }
-//
-//  @Test
-//  public void testGetDistributionViewDataWithoutBucketBoundaries() {
```

Comment:
Do you think it would be useful to test a view that has only a subset of the possible Aggregations?